projects.go

```diff
@@ -464,59 +465,82 @@ func (s *ProjectsService) GetProjectEvents(pid interface{}, opt *GetProjectEvent
 	return p, resp, err
 }
 
+// ProjectContainerExpirationPolicy represents the container expiration attributes for a project.
+type ProjectContainerExpirationPolicy struct {
+	Cadence         string    `json:"cadence"`
+	KeepN           uint      `json:"keep_n"`
+	OlderThan       string    `json:"older_than"`
+	NameRegex       string    `json:"name_regex"` // NameRegex will be replaced by NameRegexDelete but now calls without setting this field are not accepted.
```

Comment:
```suggestion
```

This struct is used to marshal the response, so it should not matter that this field doesn't exist.

Reply:
I've removed the field. Though when sending it using the Attributes struct, the fields needs to be present for the request to be accepted by Gitlab.